Terraform.sublime-syntax

```diff
@@ -247,6 +250,31 @@ contexts:
   # https://www.terraform.io/docs/language/expressions/types.html
   # https://github.com/hashicorp/hcl2/blob/master/hcl/hclsyntax/spec.md#template-expressions
   string_literals:
+    - match: (")(true)(")
```

Comment:
Highlighting booleans in strings is weird to me. These aren't actually booleans and we don't do this in other languages either. What is the rationale behind this?

Reply:
I can't provide the source I was highlighting, but there were some properties in strings that were helpful to mark. It looked something like this:

```
resource "aws_something" "name" {
    ...
    something = "true"
    ...
}
```

Like I said in the OP, I can imagine dispute of this style. I won't be offended if you take it out. (But don't just revert the PR. Leave in the extended Boolean scopes.)

Reply:
I did not PR my huge IP/CIDR ruleset, but I could. It (like the AWS ACL rules) is only applicable inside strings. At which point, why block booleans and numbers in strings?

Reply:
TBH, right after merging this one - looking at some examples, I also got somewhat skeptical about `"true"` and `"false"` being scoped constant, especially as in some examples they are explicitly used in function arguments, converting them to bools.

Reply:
Well, that is also weird, but indeed, perhaps not shipping this in the first place will be a better update UX than adding it and removing it again in the future.

Reply:
> not shipping this in the first place will be a better update UX than adding it and removing it again in the future.

Agreed. If you're shipping soon and having negative thoughts for this, I think you should take it out... at least until you're comfortable.

Reply:
Removed via 01eba127fe3411a5c53c04866172d84d294b9e59.